app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controller/app.controller");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((req, res, next) => {
+  res.status(404).send({ msg: "Endpoint not found!" });
+});
```

Comment:
I'd be wary of lines 9 - 11, when you bring in other middleware you will have to put this at the bottom as theres no if condition on line 10. By doing this you will accidentally diagnose all your unknown 500 errors to be 404. I would use:
 app.all(*, (req, res, next) => {
  res.status(404).send({ msg: "Endpoint not found!" });
}

Instead :) This will be reached if none of the above endpoints have been triggered, it will default to sending back a 404 not found :)